iceoryx_hoofs/source/log/platform_building_blocks/logger.cpp

```diff
@@ -67,7 +64,7 @@ LogLevel logLevelFromEnvOr(const LogLevel logLevel) noexcept
         }
         else
         {
-            puts("Invalide value for 'IOX_LOG_LEVEL' environment variable!'");
+            puts("Invalid value for 'IOX_LOG_LEVEL' environment variable!'");
```

Comment:
For the user it would be helpful if you would list the valid values in here. Then the user spots the error much easier otherwise one has to find the right section in our documentation which can be a challenge.